controller/api/apis.go

```diff
@@ -2882,7 +2882,14 @@ type RESTCrdDlpGroupConfig struct {
 }
 
 type RESTDlpSensorExport struct {
-	Names []string `json:"names"`
+	Names         []string           `json:"names"`
+	ExportOptions *RESTExportOptions `json:"export_options"`
+}
+
+type RESTExportOptions struct {
```

Comment:
Missing in controller/api/apis.yaml

Reply:
I will update the `apis.yaml` when all of the changes are settled again.